UN-OCHA/ocha_basic
# CD-33 search block ala Drupal

Trying to integrate search component from styleguide with Drupal's search block.
It has its own region, and a preprocess function to change placeholder, set button html and due to Drupalisms, applies a class of 'element-invisible' on the input submit, so the form still submits but using our button element.

### Testing

1. checkout branch
2. Place search block in header-search region
3. clear cache
4. check visual display
5. try a search
